app/safe/internal/bootstrap/k8s.go

```diff
@@ -21,6 +23,16 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+var podNamespace string
+var id string = "VSECMSAFE"
+func init() {
+	podNamespace = os.Getenv("POD_NAMESPACE")
```

Comment:
@abhishek44sharma one tiny request:
Instead of `POD_NAMESPACE`, can we use `VSECM_SYSTEM_NAMESPACE`.

My reasoning:
* By convention we have prefixed all of our configuration variables so far (for example: VSECM_SIDECAR_SUCCESS_THRESHOLD)
* This workload does not have to be a pod; in the future we might decide to use a DaemonSet or something else, when we consider certain HA scenarios; so better to keep “POD” out of the variable.

Reply:
Got it, make sense @v0lkan. Will make the change and re-submit for review!! 👍